Assets/MixedRealityToolkit/_Core/Extensions/NetNativeReflectionExtensions.cs

```diff
@@ -1,211 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
-
-#if !ENABLE_IL2CPP && WINDOWS_UWP
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Reflection;
-
-namespace Microsoft.MixedReality.Toolkit.Internal.Extensions
```

Comment:
Why remove?

Reply:
Most of this file was a duplicate of ReflectionExtensions.cs.  I moved the methods needed there.

Reply:
Nice catch, I didn't realize I had already ported the reflection extensions